### PR TITLE
Adds logging to superfart.

### PR DESCRIPTION
**Pull request by GuyonBroadway**
Simple enough, should make adminning superfarts a LOT easier.
:cl:
admin: Added logging to superfarts
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
thank you for this

+1

Greetings,
Fluxcapacitor1337
Head Administration

File: hippiestation/code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -203,6 +203,7 @@
 					if(M != user)
 						user.visible_message("<span class='warning'><b>[user]</b>'s ass blasts <b>[M]</b> in the face!</span>", "<span class='warning'>You ass blast <b>[M]</b>!</span>")
 						M.apply_damage(50,"brute","head")
+						log_admin("[key_name(user)] dealt superfart damage to [key_name(M)]")
```
Comment (Carbonhell) [inline hippiestation/code/modules/mob/living/carbon/human/emote.dm]: Why log admin, shouldn't it be log attack

**Review by Carbonhell — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
-1,  superfarts are a classic "fuck-you" to anybody who manages to catch you. it's hilarious and fits our hippie funloving ways

**Comment by Carbonhell** [comment]
This is a bugfix, not a feature. All sources of damage should be logged.

File: hippiestation/code/modules/mob/living/carbon/human/emote.dm (hunk 1 above)
Comment (GuyonBroadway) [reply hippiestation/code/modules/mob/living/carbon/human/emote.dm]: It was 2:30AM and I was drunk is why.

Comment (GuyonBroadway) [reply hippiestation/code/modules/mob/living/carbon/human/emote.dm]: Ok this should be better now.

**Review by Carbonhell — APPROVED** [review]

**Comment by Carbonhell** [comment]
ay don't forget changelogs next time please

**Pull request merged**